source/parser.hera

```diff
@@ -4779,6 +4779,10 @@ ReservedWord
 
 # https://262.ecma-international.org/#sec-comments
 Comment
+  # perf: assertion to exit early
+  /(?=\/|#)/ _Comment -> $2
```

Comment:
I think the regex is better because `&("/" / "#")` compiles into four function calls but `/(?=\/|#)/` compiles into a single regex test. Eventually Hera should perform some of these optimizations automatically but that might be a while.

Reply:
I agree on the first, but I assumed `&[/#]` was equivalent to `&/[\/#]/` and that this just did the regex — but maybe the `&` still incurs a function call. 🙁